apps/tasks/lib/controllers/my_tasks_controller.dart

```diff
@@ -25,6 +25,12 @@ class MyTasksController extends ChangeNotifier {
   /// The loaded tasks which have [TaskStatus.done]
   List<TaskWithPatient> get done => _tasks.where((element) => element.status == TaskStatus.done).toList();
 
+  /// all loaded patients
+  List<Patient> _allPatients = [];
+
+  List<Patient> get allPatients => _allPatients;
```

Comment:
I meant on the `PatientsByAssignmentStatus` as this would simplify the assignment in Line 46 to
```
_allPatients = patients.all
```